### PR TITLE
Partner Component

### PR DESCRIPTION
**Pull request by Viktor-creative**

### CODE REVIEW

File: src/components/navigation/BottomNavigation.vue [hunk 1]
```diff
@@ -70,4 +70,8 @@ export default {
     }
   }
 }
+.test {
```
Comment (mirkopleli) [inline src/components/navigation/BottomNavigation.vue]: https://pineco.de/css-quick-tip-the-valid-characters-in-a-custom-css-selector/#:~:text=A%20Basic%20Valid%20CSS%20Class,hyphen%20followed%20by%20a%20number.

File: src/App.vue [hunk 2]
```diff
@@ -36,4 +38,11 @@ export default {
     transition-duration: 0.4s;
   }
 }
+.navigation {
+  position: relative;
+}
+.test {
```
Comment (mirkopleli) [inline src/App.vue]: https://pineco.de/css-quick-tip-the-valid-characters-in-a-custom-css-selector/#:~:text=A%20Basic%20Valid%20CSS%20Class,hyphen%20followed%20by%20a%20number.

File: src/components/partner/Partner.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  created () {
+    partnerService.getPartnerRecords().then(list => {
+      this.partnerList = list
+    })
+  }
+}
+</script>
+
+<style scoped>
```
Comment (mirkopleli) [inline src/components/partner/Partner.vue]: Unused

File: src/components/navigation/BottomNavigation.vue (hunk 1 above)
Comment (mirkopleli) [reply src/components/navigation/BottomNavigation.vue]: Please use correct class names

File: src/components/partner/Partner.vue [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<template>
+<div>
+  <h2>Partner:</h2>
+    <v-carousel
+    cycle
+    height="200"
```
Comment (mirkopleli) [inline src/components/partner/Partner.vue]: Is it possible to make the size of the logos smaller? On desktop they are already very large

**Review by mirkopleli — CHANGES_REQUESTED** [review]

**Review by AlexanderKugler — COMMENTED** [review]
Siehst du noch Fehler ?

**Review by jonasdegener — APPROVED** [review]

**Pull request merged**